widget/embedded/src/components/TypesBlockchain /TypesBlockchain.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable @typescript-eslint/no-magic-numbers */
+import type { PropTypes } from './TypesBlockchain.types';
+
+import { i18n } from '@lingui/core';
+import { Divider, Image, Skeleton, Typography } from '@rango-dev/ui';
+import React from 'react';
+
+import { useMetaStore } from '../../store/meta';
+import { ChainsChip } from '../ChainsChip';
+
+import { generateChainsLogo } from './TypesBlockchain.helpers';
+import { Container, FirstImage, ImageContent } from './TypesBlockchain.styles';
+import { BlockchainType } from './TypesBlockchain.types';
+
+export function TypesBlockchain(props: PropTypes) {
+  const { setType, type } = props;
+  const blockchains = useMetaStore.use.meta().blockchains;
+  const loadingStatus = useMetaStore.use.loadingStatus();
+
+  return (
+    <Container>
+      {loadingStatus === 'loading'
+        ? Array.from(Array(5), (_, index) => (
+            <Skeleton
+              key={index}
+              variant="rounded"
+              height={65}
+              width={index === 0 ? 45 : 65}
+            />
+          ))
+        : Object.keys(BlockchainType).map((chainType) => (
```

Comment:
to get rid of type assertion, I suggest `const types = Object.keys(BlockchainType) as BlockchainType[];` instaed.